packages/ui/src/components/file-explorer.tsx

```diff
@@ -0,0 +1,99 @@
+import { ReactNode } from 'react'
+import { Link } from 'react-router-dom'
+
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger, Icon, Text } from '@/components'
+import { cn } from '@utils/cn'
+
+interface FolderItemProps {
+  children: ReactNode
+  value?: string
+  isActive?: boolean
+  content?: ReactNode
+  chevronClass?: string
+  link: string
+}
+
+function FolderItem({ children, value, isActive, content, chevronClass, link }: FolderItemProps) {
+  return (
+    <AccordionItem value={value || ''} className="border-none">
+      <AccordionTrigger
+        className="w-full p-0"
+        leftChevron
+        rotateChevron
+        chevronClass={chevronClass || 'text-icons-7 group-hover:text-icons-2 group-data-[state=open]:text-icons-2'}
```

Comment:
I noticed that in `CommandList` we use `heightClassName`, but here we use `chevronClass`, can we standardise on whether we include the `Name` part across components? I'd say we should use it to fall in line with `className`
```suggestion
        chevronClassName={chevronClassName || 'text-icons-7 group-hover:text-icons-2 group-data-[state=open]:text-icons-2'}
```